src/21-pubsub/index.md

```diff
@@ -0,0 +1,105 @@
+---
+# this is the final blog post's id (used in the directory)
+# this id is part of the url, and should only contain:
+#   letters, numbers, dashes.
+id: 21-pubsub
+
+breadcrumbs:
+  - {name: "Take a look at pubsub on ipfs!", link: "./" }
+
+# the date here should be set to the final publication date,
+# on the day it is published.
+date: 2016-11-09
+
+# this is the Title
+title: Take a look at pubsub on ipfs!
+
+# this is the name of the main author(s)
+author: Jeromy Johnson
+
+# technical details required for the software, don't change these.
+baseurl: ..
+template: tmpl/layouts/post.html
+collection: posts
+---
+
+We are very excited to announce that we have recently merged a simple,
+experimental pubsub implementation.  While the implementation is experimental,
+and is by no means as performant or secure as our imagined 'final'
+implementation, this new feature does allow for some very fun new applications
+to be developed. In this post I will explain how to get started using `ipfs
+pubsub` and give some example applications that can and have been built using
+it.
+
+First off, to enable the pubsub code, make sure you're running a recent version
+of go-ipfs, or try out the pre-built floodsub binaries up on [the distributions
+page](https://dist.ipfs.io/go-ipfs/floodsub-2). Once you have that version of ipfs installed, start the daemon with:
+
+```sh
+> ipfs daemon --enable-pubsub-experiment
+```
+
+This will tell ipfs to create and enable the pubsub service. It also implies that you will not be able to use pubsub with other peers who are not running with pubsub enabled.
```

Comment:
"It also implies that you will only be able to use pubsub with peers who also choose to enable it."